demo/draw-path/index.html

```diff
@@ -0,0 +1,126 @@
+    <link rel="stylesheet" href="../../examples/static/css/spectre.min.css" type="text/css">
+    <link rel="stylesheet" href="../../examples/static/css/icons.min.css" type="text/css">
+    <link rel="stylesheet" href="../../examples/static/css/main.css" type="text/css">
+    <link rel="stylesheet" href="../static/css/style.css" type="text/css">
+    <link rel="stylesheet" href="./css/path.css" type="text/css">
+    <link rel="stylesheet" href="./css/legend.css" type="text/css">
+    <link rel="stylesheet" href="./css/step.css" type="text/css">
+    <script defer src="../../examples/static/js/link-to-sources.js"></script>
+    <!-- load bpmn-visualization -->
+    <script defer src="https://cdn.jsdelivr.net/npm/bpmn-visualization@0.30.0/dist/bpmn-visualization.min.js"></script>
+    <script defer src="https://cdn.jsdelivr.net/npm/values.js@2.0.0/dist/index.umd.js" integrity="sha256-lNJnNIdh5oXecah6rOix/TxyGcOYnfkybx4Qii2uxSo="
+            crossorigin="anonymous"></script>
+    <script defer src="https://variancecharts.com/cdn/variance-noncommercial-standalone-6d26aa2.min.js" charset="UTF-8"></script>
+    <script defer src="../../examples/static/js/diagram/bpmn-diagrams.js"></script>
+    <script defer src="../static/js/use-case.js"></script>
+    <script defer src="./js/paths.js"></script>
+    <script defer src="./js/steps.js"></script>
+    <script defer src="./js/path-use-case.js"></script>
+    <script defer src="./js/index.js"></script>
+</head>
+<body>
+<header class="navbar bg-dark p-2">
+    <section class="navbar-section ml-2">
+        <a class="home" href="../../examples/index.html">
+            <img src="../../examples/static/img/logo_64x64_white.png" alt="logo" class="logo">
+            <span class="h3 text-light ml-2 lib-name-text">bpmn-visualization examples</span>
+        </a>
+    </section>
+    <section class="navbar-section">
+        <a href="https://process-analytics.github.io/bpmn-visualization-js/" class="mr-2 btn btn-secondary">Docs</a>
+        <div class="dropdown mr-2">
+            <a href="#" class="btn btn-primary dropdown-toggle" tabindex="0">
+                GitHub <span class="icon icon-caret icon-in-the-middle"></span>
+            </a>
+            <ul class="menu">
+                <li><a href="https://github.com/process-analytics/bpmn-visualization-js"
+                       class="mr-2 btn btn-link">Library</a></li>
+                <li><a href="https://github.com/process-analytics/bpmn-visualization-examples"
+                       class="mr-2 btn btn-link">Examples</a></li>
+            </ul>
+        </div>
+    </section>
+</header>
+
+<section class="container col-12 flex-centered" style="margin-top: .8rem !important">
+    <div class="col-11 col-mx-auto">
+        <div class="col-10 col-mx-auto">
+            <h2>Draw me a path</h2>
+            <p>For instance, it's only possible to visualize the path between two consecutive shapes.</p>
```

Comment:
**issue:** this is a negative only message. I suggest we switch to the positive way which explains what is possible instead.
```suggestion
            <p>Visualize the path between two consecutive shapes.</p>
```